[Bug]: Datanode CrashLoopBackOff after cluster reinstall

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: master-20220630-a803f9e4
- Deployment mode(standalone or cluster): cluster with kafka
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```


### Current Behavior

```
deploy-testing-etcd-0                               1/1     Running            0          28m
deploy-testing-etcd-1                               1/1     Running            1          28m
deploy-testing-etcd-2                               1/1     Running            1          28m
deploy-testing-milvus-datacoord-77c5554bc9-jvbdm    1/1     Running            3          28m
deploy-testing-milvus-datanode-ff6f7797b-q546r      0/1     CrashLoopBackOff   6          28m
deploy-testing-milvus-indexcoord-6b9f55568d-hqbkk   1/1     Running            2          28m
deploy-testing-milvus-indexnode-58c79ddd59-dpmpr    1/1     Running            0          28m
deploy-testing-milvus-proxy-5d6c5b9694-d9nj2        1/1     Running            2          28m
deploy-testing-milvus-querycoord-845849f579-bxtj7   1/1     Running            2          28m
deploy-testing-milvus-querynode-7b95bbb8df-z5kfm    1/1     Running            0          28m
deploy-testing-milvus-rootcoord-695c699847-6gpcj    1/1     Running            2          28m
deploy-testing-minio-0                              1/1     Running            0          28m
deploy-testing-minio-1                              1/1     Running            0          28m
deploy-testing-minio-2                              1/1     Running            0          28m
deploy-testing-minio-3                              1/1     Running            0          28m
deploy-testing-pulsar-bookie-0                      1/1     Running            0          28m
deploy-testing-pulsar-bookie-1                      1/1     Running            0          28m
deploy-testing-pulsar-bookie-init-vbdqt             0/1     Completed          0          28m
deploy-testing-pulsar-broker-0                      1/1     Running            0          [28](https://github.com/zhuwenxing/milvus/runs/7126714863?check_suite_focus=true#step:15:29)m
deploy-testing-pulsar-proxy-0                       1/1     Running            0          28m
deploy-testing-pulsar-pulsar-init-7gxtq             0/1     Completed          0          28m
deploy-testing-pulsar-recovery-0                    1/1     Running            0          28m
deploy-testing-pulsar-zookeeper-0                   1/1     Running            0          28m
```

### Expected Behavior

all pods ready

### Steps To Reproduce

```markdown
see https://github.com/zhuwenxing/milvus/runs/7126714863?check_suite_focus=true
```


### Milvus Log

failed job: https://github.com/zhuwenxing/milvus/runs/7126714863?check_suite_focus=true
log: https://github.com/zhuwenxing/milvus/suites/7157204657/artifacts/284925242

### Anything else?

_No response_

error log in datanode
```
time="2022-06-30T08:09:20Z" level=error msg="[Failed to create consumer]" consumerID=828 error="server error: ConsumerBusy: Exclusive consumer is already connected" name=jmyuk subscription=by-dev-dataNode-26-434261111975444481 topic="persistent://public/default/by-dev-rootcoord-dml_39"
time="2022-06-30T08:09:20Z" level=error msg="[Failed to create consumer]" consumerID=828 error="server error: ConsumerBusy: Exclusive consumer is already connected" name=jmyuk subscription=by-dev-dataNode-26-434261111975444481 topic="persistent://public/default/by-dev-rootcoord-dml_39"
time="2022-06-30T08:09:20Z" level=error msg="[Failed to create consumer]" consumerID=827 error="server error: ConsumerBusy: Exclusive consumer is already connected" name=ogotz subscription=by-dev-dataNode-26-434261101856423937 topic="persistent://public/default/by-dev-rootcoord-dml_26"
time="2022-06-30T08:09:20Z" level=error msg="[Failed to create consumer]" consumerID=827 error="server error: ConsumerBusy: Exclusive consumer is already connected" name=ogotz subscription=by-dev-dataNode-26-434261101856423937 topic="persistent://public/default/by-dev-rootcoord-dml_26"
panic: Failed to create consumer by-dev-rootcoord-dml_38, error = All attempts results:
attempt #1:server error: ConsumerBusy: Exclusive consumer is already connected
attempt #2:server error: ConsumerBusy: Exclusive consumer is already connected
attempt #3:server error: ConsumerBusy: Exclusive consumer is already connected
attempt #4:server error: ConsumerBusy: Exclusive consumer is already connected
attempt #5:server error: ConsumerBusy: Exclusive consumer is already connected
attempt #6:server error: ConsumerBusy: Exclusive consumer is already connected
attempt #7:server error: ConsumerBusy: Exclusive consumer is already connected
attempt #8:server error: ConsumerBusy: Exclusive consumer is already connected
attempt #9:server error: ConsumerBusy: Exclusive consumer is already connected
attempt #10:server error: ConsumerBusy: Exclusive consumer is already connected
attempt #11:server error: ConsumerBusy: Exclusive consumer is already connected


goroutine 210 [running]:
github.com/milvus-io/milvus/internal/mq/msgstream.(*MqTtMsgStream).AsConsumerWithPosition(0xc005ed5400, 0xc002b33170, 0x1, 0x1, 0xc0022a8660, 0x25, 0x1)
	/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/mq_msgstream.go:657 +0x2df
github.com/milvus-io/milvus/internal/mq/msgstream.(*MqTtMsgStream).AsConsumer(0xc005ed5400, 0xc002b33170, 0x1, 0x1, 0xc0022a8660, 0x25)
	/go/src/github.com/milvus-io/milvus/internal/mq/msgstream/mq_msgstream.go:626 +0x66
github.com/milvus-io/milvus/internal/datanode.newDmInputNode(0x3192110, 0xc000cbb180, 0xc004aa62a0, 0xc0033befa0, 0xc003100b40, 0x2c, 0x5df)
	/go/src/github.com/milvus-io/milvus/internal/datanode/flow_graph_dmstream_input_node.go:47 +0x2db
github.com/milvus-io/milvus/internal/datanode.(*dataSyncService).initNodes(0xc000a689c0, 0xc00652bee0, 0x3192110, 0xc000cbb180)
	/go/src/github.com/milvus-io/milvus/internal/datanode/data_sync_service.go:260 +0x169c
github.com/milvus-io/milvus/internal/datanode.newDataSyncService(0x3192110, 0xc0009c1a80, 0xc000588000, 0xc001ae28a0, 0x31c2c90, 0xc001ae2840, 0x3182aa8, 0xc002590840, 0x7f020ccb3b18, 0xc0009ccb40, ...)
	/go/src/github.com/milvus-io/milvus/internal/datanode/data_sync_service.go:95 +0x23f
github.com/milvus-io/milvus/internal/datanode.(*flowgraphManager).addAndStart(0xc0009ccc00, 0xc00094cc00, 0xc00652bee0, 0x2, 0x2)
	/go/src/github.com/milvus-io/milvus/internal/datanode/flow_graph_manager.go:58 +0x890
github.com/milvus-io/milvus/internal/datanode.(*DataNode).handlePutEvent(0xc00094cc00, 0xc001025880, 0x3, 0xc001ae2120, 0x0)
	/go/src/github.com/milvus-io/milvus/internal/datanode/data_node.go:377 +0x2cb
github.com/milvus-io/milvus/internal/datanode.(*channelEventManager).retryHandlePutEvent(0xc000593800, 0x1, 0xc004284ea0, 0x2c, 0x3, 0xc001025880)
	/go/src/github.com/milvus-io/milvus/internal/datanode/event_manager.go:132 +0x6fb
github.com/milvus-io/milvus/internal/datanode.(*channelEventManager).Run.func1(0xc000593800)
	/go/src/github.com/milvus-io/milvus/internal/datanode/event_manager.go:70 +0x165
created by github.com/milvus-io/milvus/internal/datanode.(*channelEventManager).Run
	/go/src/github.com/milvus-io/milvus/internal/datanode/event_manager.go:64 +0x3f

```

--- a/.github/workflows/deploy-test.yaml
+++ b/.github/workflows/deploy-test.yaml
@@ -227,7 +227,10 @@ jobs:
         run: |
           yq -i ".kafka.enabled = false" cluster-values.yaml
           yq -i ".pulsar.enabled = false" cluster-values.yaml
+          yq -i ".kafka.enabled = false" cluster-values-second.yaml
+          yq -i ".pulsar.enabled = false" cluster-values-second.yaml
           yq -i ".${{ matrix.mq_type }}.enabled = true" cluster-values.yaml
+          yq -i ".${{ matrix.mq_type }}.enabled = true" cluster-values-second.yaml
           if [ ${{ matrix.mq_type }} == "kafka" ]; then
               yq -i ".kafka.enabled = true" standalone-values.yaml;
           fi

